README.md

```diff
@@ -8,3 +8,312 @@ To build and run this application please use the `kraft` script:
 
 For more information about `kraft` type ```kraft -h``` or read the
 [documentation](http://docs.unikraft.org).
+
+## Requirements
+In order to set up, configure, build and run Duktape on Unikraft, the following packages are required:
+
+* `build-essential` / `base-devel` / `@development-tools` (the meta-package that includes `make`, `gcc` and other development-related packages)
+* `sudo`
+* `flex`
+* `bison`
+* `git`
+* `wget`
+* `uuid-runtime`
+* `qemu-system-x86`
+* `qemu-system-arm`
+* `qemu-kvm`
+* `sgabios`
+* `gcc-aarch64-linux-gnu`
+
+GCC >= 8 is required to build Duktape on Unikraft.
+
+On Ubuntu/Debian or other `apt`-based distributions, run the following command to install the requirements:
+
+```console
+apt-get install -y --no-install-recommends \
+    build-essential \
+    sudo \
+    gcc-aarch64-linux-gnu \
+    libncurses-dev \
+    libyaml-dev \
+    flex \
+    bison \
+    git \
+    wget \
+    uuid-runtime \
+    qemu-kvm \
+    qemu-system-x86 \
+    qemu-system-arm \
+    sgabios
+```
+
+In addition, Python 2.7 needs to be installed under the name "python2" with pyYaml. This can be done with the following bash commands:
```

Comment:
```suggestion
In addition, Python 2.7 needs to be installed under the name `python2` with pyYaml. This can be done with the following bash commands:
```